docs/SetupGuide_Python.md

```diff
@@ -115,23 +113,27 @@ Note: This tutorial requires that a SQL database is setup as shown in [Create a
 
 #### Query String
 
-_TODO_
+The input binding executes the `SELECT * FROM Products WHERE Cost = @Cost` query. The Parameters argument passes the `{cost}` specified in the URL that triggers the function, `getproducts/{cost}`, as the value of the `@Cost` parameter in the query. CommandType is set to `Text`, since the constructor argument of the binding is a raw query.
```

Comment:
As I mentioned to Maddy in the other one - it'd be better I think to put these descriptions into the functions themselves and then just have this be the link to the function folder. That way people looking at the samples directly can benefit from the descriptions too.